distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java

```diff
@@ -13,71 +13,110 @@
+                  return false;
+                }
+              }
+              return true;
+            });
+
+    return oldProps;
   }
 
-  public static void configureFilterInvalidMetacards(
-      String filterErrors, String filterWarnings, AdminConfig configAdmin) throws IOException {
-    Configuration config =
-        configAdmin.getConfiguration(METACARD_VALIDITITY_FILTER_PLUGIN_SERVICE_PID, null);
-
+  // This method is intended to be used to configure the service for a test.
+  public static Dictionary<String, Object> configureMetacardValidityFilterPlugin(
+      List<String> securityAttributeMappings,
+      boolean filterErrors,
+      boolean filterWarnings,
+      AdminConfig configAdmin)
+      throws IOException {
     Dictionary<String, Object> properties = new DictionaryMap<>();
-    properties.put("filterErrors", filterErrors);
-    properties.put("filterWarnings", filterWarnings);
-    config.update(properties);
+    properties.put("attributeMap", securityAttributeMappings);
+    properties.put("filterErrors", String.valueOf(filterErrors));
+    properties.put("filterWarnings", String.valueOf(filterWarnings));
+    return configureMetacardValidityFilterPlugin(properties, configAdmin);
   }
 
-  public static void configureEnforceValidityErrorsAndWarnings(
-      String enforceErrors, String enforceWarnings, AdminConfig configAdmin) throws IOException {
-    Configuration config =
-        configAdmin.getConfiguration(METACARD_VALIDITITY_MARKER_PLUGIN_SERVICE_PID, null);
-
-    Dictionary<String, Object> properties = new DictionaryMap<>();
-    properties.put("enforceErrors", enforceErrors);
-    properties.put("enforceWarnings", enforceWarnings);
-    config.update(properties);
+  // This method should only be used to reset a configuration using the Dictionary returned in
+  // the overloaded method.
+  public static Dictionary<String, Object> configureMetacardValidityFilterPlugin(
+      Dictionary<String, Object> props, AdminConfig configAdmin) throws IOException {
+    return configureService(METACARD_VALIDITY_FILTER_PLUGIN_SERVICE_PID, props, configAdmin);
   }
 
-  public static void configureEnforcedMetacardValidators(
-      List<String> enforcedValidators, AdminConfig configAdmin) throws IOException {
-
-    // Update metacardMarkerPlugin config with no enforcedMetacardValidators
-    Configuration config =
-        configAdmin.getConfiguration(METACARD_VALIDITITY_MARKER_PLUGIN_SERVICE_PID, null);
-
+  // This method is intended to be used to configure the service for a test.
+  public static Dictionary<String, Object> configureValidationMarkerPlugin(
+      List<String> enforcedValidators,
+      boolean enforceErrors,
+      boolean enforceWarnings,
+      AdminConfig configAdmin)
+      throws IOException {
     Dictionary<String, Object> properties = new DictionaryMap<>();
     properties.put("enforcedMetacardValidators", enforcedValidators);
-    config.update(properties);
```

Comment:
I don't think we should be doing String.valueOf(...).  The boolean values should go into config admin as booleans.  I see this in a few places in this file.

Reply:
Interesting why it works both ways. I'll change it

Reply:
It does because of blueprint which will parse them before calling the managed object setters. However, Dan is right, config should be in the format described by the metatype